tests/integration_tests/tests/agentless_tests/test_resume.py

```diff
@@ -329,3 +335,44 @@ def test_force_resume_cancelled_nonresumable(self):
                         .runtime_properties['resumed'])
         self.assertTrue(self.client.node_instances.get(instance2.id)
                         .runtime_properties['marked'])
+
+    def test_restart(self):
+        """Test that the restart builtin workflow can be resumed.
+
+        We'll test resuming at both the "stop" stage and the "start" stage.
```

Comment:
I prefer for tests debugging to separate this test to two

Reply:
i dont want to do that because that makes the test much weaker. Currently it's the only test that resumes 2 separate graphs in a single execution; if it was separate then it'd be 2 tests that resume just one graph, and we already have tests that do that